packages/ceramic-doctype-tile/src/tile-doctype.ts

```diff
@@ -105,15 +111,33 @@ export class TileDoctype extends Doctype {
             header.owners = newOwners
         }
 
+        header.nonce = TileDoctype._calculateNonce(doctype)
```

Comment:
Only set this if the calculated nonce > 0?

Reply:
I can omit `0` but I set it here to `0` for consistency. `genesis` and `anchor` records don't have `nonce`s. Makes sense?

Reply:
Right, seems like it's just adding overhead for no real benefit though?

Reply:
I wanted to avoid some future places in CAS where I would need to query if `nonce === 0 || nonce === undefined`. I'll revert it for now.